microk8s-resources/wrappers/run-containerd-with-args

```diff
@@ -28,7 +28,17 @@ if [ -e "${SNAP_DATA}/var/lock/gpu" ]; then
 else
   RUNTIME="runc"
 fi
-sed 's@${SNAP}@'"${SNAP}"'@g;s@${SNAP_DATA}@'"${SNAP_DATA}"'@g;s@${RUNTIME}@'"${RUNTIME}"'@g' $SNAP_DATA/args/containerd-template.toml > $SNAP_DATA/args/containerd.toml
+
+# Determine the underlying filesystem that containerd will be running on
+FSTYPE=$(/usr/bin/stat -f -c %T "${SNAP_COMMON}")
```

Comment:
Here you are using `/usr/bin/stat` but do we know this will exist on the host machine? We know that stat is packaged with the snap. It is better that we do not set the full path to stat here and let it resolve based on the $PATH set. According to the PATH set above the in-snap stat will be used.

Reply:
I agree. I will amend the code to just call stat and based on the PATH being set above, use the stat packaged with the snap.